### ISSUE TITLE
Consider moving mkchromecast to the System Status Menu

### ISSUE THREAD

**Issue by pwd-github** [muammar/mkchromecast#223]
With the icon tray removed, there was a proposal to create an app, but the cleaner solution would be just to move it to the System Status Menu, as that's actually a better fit for the functionality.

**Comment by leamas** [comment]
>With the icon tray removed, there was a proposal to create an app, but the cleaner solution would be just to move it to the System Status Menu, as that's actually a better fit for the functionality.

Perhaps on some platform (which do you refer to?). But  I'd guess that a standalone application is the only viable cross-platform solution. Please correct me if I'm wrong.

**Comment by pwd-github** [comment]
Sorry, this was in reference to the GNOME (Linux) issue. The standalone app, while a viable cross platform solution, is terrible from a UX standpoint.

**Comment by muammar** [comment]
I agree with @pwd-github. The only reason Mkchromecast became used was due to the system tray icon... After reading https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome/, I kind of understand the reason why Gnome decided to get rid of their system tray -- it was a shitty implementation. But there is https://extensions.gnome.org/extension/495/topicons/ extension that should come handy. 

Now I have a doubt: what is the difference between a **System Status Menu** and a **System Tray**? I could not find anything yet on the PyQt documentation unless I am looking at the wrong terms.

**Comment by pwd-github** [comment]
So the docs for the System Status Menu are here: https://wiki.gnome.org/Design/OS/SystemStatus

And an example of an app with a similar implementation to what I'm suggesting is here: https://extensions.gnome.org/extension/906/sound-output-device-chooser/
With another being here:
https://extensions.gnome.org/extension/116/hide-volume-button/
